fvm/state/state.go

```diff
@@ -353,10 +358,14 @@ func IsFVMStateKey(owner string, key string) bool {
 		key == AccountStatusKey
 }
 
+func IsSlabIndex(key string) bool {
+	return len(key) == 9 && key[0] == '$'
```

Comment:
Can we take this function from cadence somewhere. It feels like the FVM shouldn't be responsible for knowing this.

Reply:
i did a grep on the cadence code base, it seems like the only place that does a similar check is
`./runtime/cmd/decode-state-values/main.go:	return len(key) == slabKeyLength && key[0] == '$'`

can you point me to a function if there is one?

Reply:
Hmm... I cant find anything either. I'll open a cadence issue for this.